src/main/java/io/quarkus/status/IssuesService.java

```diff
@@ -21,12 +22,14 @@
     public static final String BUG_LABEL = "kind/bug";
     private static final String ENHANCEMENT_NAME = "Enhancements";
     public static final String ENHANCEMENT_LABEL = "kind/enhancement";
-    private static final LocalDate ISSUES_STATS_START = LocalDate.of(2019, 1, 1);
     private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM");
 
     @Inject
     GitHubService gitHubService;
 
+    @ConfigProperty(name = "quarkus-status.issues.stats.start", defaultValue = "2019-01-01")
```

Comment:
```suggestion
    @ConfigProperty(name = "status.issues.stats.start", defaultValue = "2019-01-01")
```

That's how the other is called (the token).